modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,198 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// (link:https://issues.redhat.com/browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you use`tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
+// (link:https://issues.redhat.com/browse/SRVKP-1758[SRVKP-1758 Pipeline as code: validate repository URL uniqueness])
+// Shivam Mukhade
+
+* This update implements `Step` and `Sidecar` overrides for `TaskRun` resources.
+// link:https://github.com/tektoncd/pipeline/pull/4598
+
+* This update adds minimal `TaskRun` and `Run` statuses within `PipelineRun` statuses.
+// link:https://github.com/tektoncd/pipeline/pull/4739
+
+* With this update, the graceful termination of pipeline runs feature is promoted from being a beta feature to being a stable feature. As a result, the previously deprecated `PipelineRunCancelled` status remains deprecated and is planned to be removed in a future release.
+// link:https://github.com/tektoncd/pipeline/pull/4668
+
+* With this update, you can specify the workspace for a pipeline task by using the name of the workspace. This change makes it easier to specify a shared workspace for a pair of `Pipeline` and `PipelineTask` resources. You can also continue to map workspaces explicitly.
+// link:https://github.com/tektoncd/pipeline/pull/4887
+
+* With this update, parameters in embedded specifications are propagated without mutations.
+//https://github.com/tektoncd/pipeline/pull/4845
+
+* With this update, you can specify the required metadata of a `Task` resource referenced by a `PipelineRun` resource by using annotations and labels. This way, `Task` metadata that depends on the execution context is available during the pipeline run.
+// link:https://github.com/tektoncd/pipeline/pull/4834
+
+* This update adds support for object or dictionary types in `params`  and `results` values. This change affects backward compatibilities and sometimes breaks forward compatibility, such as using an earlier client with a later {pipelines-title} version.
+Action required: This update changes the `ArrayOrStruct` structure, which affects the project using the go API as a library.
+// TBD what action?
+// link:https://github.com/tektoncd/pipeline/pull/4786
+
+* This update adds a `SkippingReason` value to the `SkippedTasks` field of the `PipelineRun` status fields so that users know why a given PipelineTask was skipped.
+// link:https://github.com/tektoncd/pipeline/pull/4829
+
+* As an alpha feature, this update supports using an `array` type for emitting results from a `Task` object. The type of the result is changed from `string` to `ArrayOrString`.
```

Comment:
```suggestion
* As an alpha feature, this update supports using an `array` type for emitting results from a `Task` object. The type of the result is changed from `string` to `ArrayOrString`.
```
- Antecedent issue with the opening phrase -- makes it sound like `this update` is an `alpha feature`.
-

Reply:
Good catch. Thanks. Updated.